hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/DBDefinitionFactory.java

```diff
@@ -98,6 +102,56 @@ public static DBDefinition getDefinition(Path dbPath,
     return getDefinition(dbName);
   }
 
+  public static List<CheckedSupplier<DBDefinition, Exception>> getFactories(
```

Comment:
This can be package-private.

```suggestion
  static List<CheckedSupplier<DBDefinition, Exception>> getFactories(
```